webapp/views/partials/includes/banner.jade

```diff
@@ -0,0 +1,15 @@
+nav(class='navbar navbar-inverse navbar-fixed-top')
+    div(class='container-fluid')
+        div(class='navbar-header')
+            button(type='button', class='navbar-toggle collapsed', data-toggle='collapse', data-target='#header-bar')
+                span(class='sr-only') Toggle navigation
+                span(class='icon-bar')
+                span(class='icon-bar')
+            a(class='navbar-brand')
+        div(class='collapse navbar-collapse', id='header-bar')
+            ul(class='nav navbar-nav')
+                li
+                    a(href='#', ng-click='goHome()') Home
+                li
+                    a(href='#', ng-click='goUsers()') Users
+            button(type='button', class='btn-logout btn btn-deafult navbar-btn', ng-click='logout()') Logout
```

Comment:
Also @paymog does that mean I should maybe define a logout state that logs out the user and redirects to mainPage? instead of defining a function that all controllers must call?